client/web/src/components/diff/FileDiffNode.tsx

```diff
@@ -48,6 +50,7 @@ export const FileDiffNode: React.FunctionComponent<FileDiffNodeProps> = ({
     className,
     extensionInfo,
     persistLines,
+    diffMode,
```

Comment:
if you initialize the default here, it's simpler to grasp the value when checking out the component :) 

```suggestion
    diffMode = 'unified',
```